config/install.yaml

```diff
@@ -12302,3 +12302,72 @@ spec:
       volumes:
       - emptyDir: {}
         name: sigstore
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/component: kyverno
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/name: kyverno
+    app.kubernetes.io/part-of: kyverno
+    app.kubernetes.io/version: latest
+  name: allow-apiserver
+  namespace: kyverno
+spec:
+  egress:
+  - ports:
+    - port: 9443
+    to:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+    - podSelector:
+        matchLabels:
+          component: kube-apiserver
+          tier: control-plane
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+    - podSelector:
+        matchLabels:
+          component: kube-apiserver
+          tier: control-plane
+    ports:
```

Comment:
The `namespaceSelector` and `podSelector` need to be in same array item, otherwise you are allowing everything from `kube-system` or `kube-apiserver` from same namespace as Kyverno